### DISCUSSION TITLE
Error saat di deploy

### DISCUSSION THREAD

**Discussion by IsmailHusun19** [deaafrizal/forum#75] [Troubleshooting Room]
Permisi mau nanya, saat di deploy halaman utama bisa diakses tapi klo halamanutama/login misalnya itu eror 404 gt, tapi klo di run di local pakai npm run dev ataupun npm run preview itu aman, kira2 knp ya
link : https://ismail-husun19-spk-pencarian-hp-terbaikk.vercel.app/

**Comment by IsmailHusun19** [comment]
![WhatsApp Image 2024-03-29 at 01 03 22](https://github.com/deaaprizal/forum/assets/128096225/04265717-b057-4383-94af-2a58d5bdfb7c)
saya pake react js dan vite

**Comment by seporang** [comment]
coba ss kode erorrnya
